libindy/Cargo.toml

```diff
@@ -44,7 +44,7 @@ log = "0.3.7"
 openssl = { version = "0.9.11", optional = true }
 rand = "0.3"
 rlp = "0.2.0"
-rusqlite = "0.10.1"
+rusqlite = { git = "https://github.com/mikelodder7/rusqlcipher", features=["bundled"] }
```

Comment:
Link to repository doesn't acceptable, we need for separate Rust crates like `rusqlcipher`
@mikelodder7 could you create it, please?